1374.generate-a-string-with-characters-that-have-odd-counts.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/generate-a-string-with-characters-that-have-odd-counts/
+ */
+
+/**
+ * @param {number} n
+ * @return {string}
+ */
+var generateTheString = function(n) {
+    
+    let alphabet1 = "a";
+    let alphabet2 = "b";
```

Comment:
```suggestion
    const alphabet1 = "a";
    const alphabet2 = "b";
```
Const is better here